### PR TITLE
[BD-46] chore(deps): update dependency paragon to v20.19.0

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
**What changed?**

- updated Paragon dependency to v20.19.0;
- updated `@edx/frontend-component-header` dependency to v3.2.1;
- updated `@edx/frontend-platform` dependency to v2.6.2;
- updated snapshot tests.

**Developer Checklist**
- [x] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: package.json [hunk 1]
```diff
@@ -53,7 +51,7 @@
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "react-helmet": "^6.1.0",
-    "react-intl": "^2.9.0",
```
Comment (ihor-romaniuk) [inline package.json:56]: You update this package on 3 major versions. Did you make sure that the translation works as previously?

Comment (PKulkoRaccoonGang) [reply package.json:56]: After raising the translation version, I didn't notice any visible changes in the MFE Profile. I checked at your request

**Review by ihor-romaniuk — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-gradebook&entry.867055334=https://github.com/openedx/frontend-app-gradebook/pull/284&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2022-11-22+21:54&entry.1917517419=2023-01-09+17:35&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**